internal/cli/server/flags.go

```diff
@@ -167,6 +169,12 @@ func (c *Command) Flags(config *Config) *flagset.Flagset {
 		Value:   &c.cliConfig.Heimdall.URL,
 		Default: c.cliConfig.Heimdall.URL,
 	})
+	f.DurationFlag(&flagset.DurationFlag{
+		Name:    "bor.heimdalltimeout",
+		Usage:   "Timeout of Heimdall service",
```

Comment:
Let's make it more descriptive as it may mean something else as well. 

Maybe something like "Timeout for outgoing requests to heimdall" (just a suggestion, if there's anything better, feel free to use it).